introduction_to_amazon_algorithms/jumpstart-foundation-models/text-generation-benchmarking/inference-benchmarking-container-example.ipynb

```diff
@@ -0,0 +1,153 @@
+{
```

Comment:
<p>Introduction?</p><p> </p><p>Also, Is this supposed to be exposed to customers? Is it supposed to show that JumpStart is in parity with other possible servings? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4443/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>